build.sbt

```diff
@@ -18,7 +18,7 @@ import scala.xml.transform.{RewriteRule, RuleTransformer}
 // Warning: 2.12.13 + crossVersion break sbt-scoverage: https://github.com/scoverage/sbt-scoverage/issues/319
 val scala212 = "2.12.10"
 val scala213 = "2.13.10"
-val defaultScalaV = scala213
+val defaultScalaV = scala212
```

Comment:
Please revert that - you can add NUSSKNACKER_SCALA_VERSION=2.12 system env variable instead of changing this file